### DISCUSSION TITLE
Private App

### DISCUSSION THREAD

**Discussion by vikingcodes** [amzn/selling-partner-api-models#3346] [General]
@chapmanjw @ShivikaK 

We run Amazon Training & Launch Program.

We need download some report from customer's account like inventory and order reports on daily basis and integrate into our database.

But we don't want launch our app on app store for public because our app designed for some specific seller who onboard on our platform.

We have any option to create private app ? Or any other options ?

**Comment by npatten** [comment]
@vikingcodes - I believe you can self authenticate for a private app. 
see: https://developer-docs.amazon.com/sp-api/docs/self-authorization
